src/utils.ts

```diff
@@ -586,3 +586,7 @@ export const filterObjectKeys = (
     .reduce((cur, key) => {
       return Object.assign(cur, { [key]: obj[key] });
     }, {});
+
+export const isKillSwitchOn = () => !!process.env['LUMIGO_SWITCH_OFF'];
+
+export const isLambdaTraced = () => !!(!isKillSwitchOn() && isAwsEnvironment());
```

Comment:
both of these methods should return a boolean value, so we should not need `!!`. for increased readability, please put the positive check first (ie. `=> (isAwsEnvironment() && !isKillSwitchOn())`)

Reply:
done